src/Microsoft.DocAsCode.Build.ManagedReference/BuildOutputs/ApiBuildOutput.cs

```diff
@@ -241,15 +246,22 @@ private static ApiBuildOutput FromModel(ItemViewModel model, Dictionary<string,
                 Attributes = model.Attributes,
                 Metadata = metadata.Concat(model.Metadata.Where(p => !metadata.Keys.Contains(p.Key))).ToDictionary(p => p.Key, p => p.Value),
             };
+            output.DerivedClasses = GetReferenceList(model.DerivedClasses, references, model.SupportedLanguages, true, output.Level+1);
+            return output;
         }
 
         private static List<ApiReferenceBuildOutput> GetReferenceList(List<string> uids,
                                                                       Dictionary<string, ApiReferenceBuildOutput> references,
                                                                       string[] supportedLanguages,
-                                                                      bool extractIndex = false)
+                                                                      bool extractIndex = false,
+                                                                      int level = -1)
```

Comment:
I think it make sense to add the new logic in the method. Need extract index?  -> yes -> provide the index or compute index ? -> provide the index...

---
In reply to: [91428041](https://github.com/dotnet/docfx/pull/973#discussion_r91428041) [](ancestors = 91428041,91427831)